accelerators/federated-learning/automated_provisioning/ps/Utils.ps1

```diff
@@ -0,0 +1,41 @@
+function Wait-For-Successful-Registration(){
```

Comment:
Nit: PowerShell method names should look like `Verb-OtherStuff`, e.g. below. Ditto for the other method names in this file.

```powershell
Wait-SuccessfulRegistration
```

These articles provide helpful context:

- [Approved Verbs for PowerShell Commands](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.2)
- [Strongly Encouraged Development Guidelines](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/strongly-encouraged-development-guidelines?view=powershell-7.2)